components/tidb_query_datatype/src/codec/chunk/column.rs

```diff
@@ -73,7 +73,15 @@ impl Column {
         let eval_type = box_try!(EvalType::try_from(field_type.as_accessor().tp()));
         match eval_type {
             EvalType::Int => {
-                if field_type.is_unsigned() {
+                if field_type.as_accessor().tp() == FieldTypeTp::Bit {
+                    if field_type.as_accessor().flen() > 64 {
+                        unimplemented!()
+                    }
+                    let start_idx: usize = ((64 - field_type.as_accessor().flen()) / 8) as usize;
```

Comment:
Is there any document on `Bit` vector (and scalar)'s encoding? These operations that doesn't look straightforward to me, and I guess it's a kind of hack to make this work.

Reply:
Bit is defined bits range from 1 to 64:  https://dev.mysql.com/doc/refman/8.0/en/bit-type.html
For higher version of mysql, Bit will be more bytes, but TiDB has't support yet，and TiDB defined Bit storage len be 8 bytes（tidb/parser/types/field_type.go +352）now，hard code； So，for future TiDB that Bit support more bytes，we should fix TiKV codes here；
Maybe I should has some comments here.

Reply:
Comments are needed for this line of code.

Reply:
ok, has added it.